ui/cypress/e2e/explorer.test.ts

```diff
@@ -438,7 +438,7 @@ describe('DataExplorer', () => {
       })
     })
 
-    it('shows flux signatures', () => {
+    it.skip('shows flux signatures', () => {
```

Comment:
Flaky test?

Reply:
Yeah, also described in more detail in the PR description. For some reason running the tests in a VM with a docker container made this test fail a lot - [the equivalent test code](https://github.com/influxdata/ui/blob/a131cdbe9cd541634b0e97ad54ce35ff098ee62c/cypress/e2e/shared/explorer.test.ts#L440) is skipped in the `ui` repo as well.